src/stylized/BulletListStyle.tsx

```diff
@@ -14,6 +13,6 @@ const BulletListStyle = (props: SvgProps): React.Element<*> => (
     <circle cx="10" cy="110" r="8" />
     <rect x="25" y="105" rx="5" ry="5" width="220" height="10" />
   </Holder>
-)
+);
 
-export default BulletListStyle
+export default BulletListStyle;
```

Comment:
I think it's a great time to remove the export from here and put in the definition of component, like:

```tsx
export default (props: ISvgProps) => (
```

Reply:
done